pkg/healthcheck/healthcheck.go

```diff
@@ -680,9 +680,10 @@ func (hc *HealthChecker) allCategories() []Category {
 					fatal:       true,
 					check: func(ctx context.Context) (err error) {
 						hc.uuid, hc.linkerdConfig, err = hc.checkLinkerdConfigConfigMap(ctx)
-						if hc.linkerdConfig != nil {
-							hc.CNIEnabled = hc.linkerdConfig.GetGlobal().CNIEnabled
+						if hc.linkerdConfig == nil {
+							return errors.New("failed to load linkerd-config")
```

Comment:
Does this actually refer to a resource called `linkerd-config`?

Reply:
yes